Userland/Utilities/sed.cpp

```diff
@@ -371,9 +371,6 @@ struct YArguments {
             return false;
         });
 
-        dbgln("Characters  = '{}'", characters);
```

Comment:
Please amend the commits that added the `dbgln()` to not add them in the first place, instead of removing them in a later commit.

(Easiest way to do this: `git reset HEAD^` to remove this commit, then `git add -p` to add just the dbgln() removals for one previous commit, `git commit -m 'fixup 1'`, `git add -p` to add fixup for 2nd commit, `git commit -m 'fixup 2'`, repeat until `git diff` no longer prints anything, then `git rebase -i master`, move the lines with the fixups below the commits they belong to, and put `f` in front of them. then they'll be merged into the previous commit.)

Reply:
Done. Thanks for guiding me through this `git` foo.